packages/webpack/src/plugin.ts

```diff
@@ -88,7 +88,8 @@ export class AotPlugin {
     }
 
     const tsConfig = tsc.readConfiguration(options.tsConfigPath, basePath);
-    this._rootFilePath = tsConfig.parsed.fileNames;
+    this._rootFilePath = tsConfig.parsed.fileNames
+      .filter(fileName => fileName.indexOf('.spec.ts') < 0);
```

Comment:
to be 100% sure: `.filter(fileName => fileName.match(/.spec.ts$/)  === null);` ? :)